src/app/components/form/form.component.html

```diff
@@ -1,20 +1,68 @@
-<form [formGroup]="jobForm">
-    <mat-form-field>
-        <mat-label>Plaats</mat-label>
-        <input matInput formControlName="plaats">
-    </mat-form-field>
-
-    <mat-form-field>
-        <mat-label>Afstand</mat-label>
-        <input matInput formControlName="afstand">
-    </mat-form-field>
-
-    <mat-form-field>
-        <mat-label>Sleutelwoorden</mat-label>
-        <input matInput formControlName="sleutelwoorden">
-    </mat-form-field>
-</form>
-
-<button (click)="submit()" mat-raised-button color="accent">
-    Search request
-</button>
+<div class="container">
+    <div fxLayout="row" fxLayoutAlign="center center" class="content">
+        <mat-card class="card" >
+            <mat-card-title fxLayout.gt-xs="row" fxLayout.xs="column">
+                <h3>Search vacancies</h3>
+            </mat-card-title>
+            <mat-card-content>
+                <div>
+                    <form [formGroup]="jobForm">
+                        <mat-form-field>
+                            <mat-label>Plaats</mat-label>
+                            <input matInput formControlName="plaats" color="white">
+                        </mat-form-field>
+
+                        <mat-form-field>
+                            <mat-label>Afstand</mat-label>
+                            <input matInput formControlName="afstand">
+                        </mat-form-field>
+
+                        <mat-form-field>
+                            <mat-label>Sleutelwoorden</mat-label>
+                            <input matInput formControlName="sleutelwoorden">
+                        </mat-form-field>
+                    </form>
+
+                    <button (click)="submit()" mat-raised-button color="accent">
+                        Search request
+                    </button>
+                </div>
+                <div class="table-container">
+                    <mat-progress-bar mode="indeterminate" *ngIf="submitted"></mat-progress-bar>
+                    <table *ngIf="dataSource!==undefined" mat-table [dataSource]="dataSource" class="mat-elevation-z8">
```

Comment:
This can link directly to the populated variable.

```suggestion
                    [dataSource]="element_data"
```
or if you applied a later review comment:
```suggestion
                    [dataSource]="vacancies"
```